src/platforms/common/configuration/options.mdx

```diff
@@ -96,6 +96,12 @@ This variable controls the total amount of breadcrumbs that should be captured.
 
 </ConfigKey>
 
+<ConfigKey name="max-cache-items" supported={[ "dotnet", "android"]} >
+
+The maximum number of [envelopes](https://develop.sentry.dev/sdk/envelopes/) to keep in cache. The SDKs use envelopes to send data, as events, attachments, user feedback, session, etc. to Sentry. An envelope can contain multiple items - for example, an event with a session and two attachments. Depending on the usage of the SDK, the size of an envelope can differ. When the number of envelopes in the local cache exceeds max-cache-items, the SDK deletes the oldest envelope. When deleting an envelope, we migrate sessions to the next envelope to not mess up your release health stats. The default is `30`.
```

Comment:
```suggestion
The maximum number of [envelopes](https://develop.sentry.dev/sdk/envelopes/) to keep in cache. The SDKs use envelopes to send data, as events, attachments, user feedback, session, etc. to Sentry. An envelope can contain multiple items - for example, an event with a session and two attachments. Depending on the usage of the SDK, the size of an envelope can differ. When the number of envelopes in the local cache exceeds max-cache-items, the SDK deletes the oldest envelope. The default is `30`.
```

I think this is implementation details, if we want to explain it, it'd be better to release health docs

Reply:
I added it because people might think that this messes up their session stats. Just wanted to clarify that this is not affected. @bruno-garcia, what do you think? Should we keep it or remove it?